src/workspaces/NewWorkspaceModal/NewWorkspaceModal.ts

```diff
@@ -346,8 +347,15 @@ const NewWorkspaceModal = withDisplayName(
       return true;
     };
 
+    const cloningGcpProtectedWorkspace =
```

Comment:
Related to this Slack discussion: https://broadinstitute.slack.com/archives/C03GMG4DUSE/p1701871209510489